lib/ensemble.lua

```diff
@@ -94,7 +94,14 @@ function Ensemble:affect(action, index, values)
     self.source_positions = values
   elseif action == actions.apply_effect then
     print('[Ensemble:affect] Applying effect to sequence:', index)
-    local performer = self.performers[parameters:get_performer(index)]
+    local performer
+    
+    if index == 5 and params:get('marco_wrong_stop') == 1 then
+      performer = self.performers['W/Tape']
```

Comment:
This is not likely to work as expected, given performers are numerically indexed. A more sophisticated lookup will be required that matches the performer name in the table.